addons/mail/static/src/models/emoji_view.js

```diff
@@ -14,12 +14,12 @@ registerModel({
             if (!this.emojiGridItemViewOwner.emojiGridRowViewOwner) {
                 return;
             }
-            if (this.emojiGridItemViewOwner.emojiGridRowViewOwner.emojiGridViewOwner.emojiPickerViewOwner.popoverViewOwner.messageActionViewOwnerAsReaction) {
-                this.emojiGridItemViewOwner.emojiGridRowViewOwner.emojiGridViewOwner.emojiPickerViewOwner.popoverViewOwner.messageActionViewOwnerAsReaction.onClickReaction(ev);
+            if (this.popoverViewTopOwner.messageActionViewOwnerAsReaction) {
+                this.popoverViewTopOwner.messageActionViewOwnerAsReaction.onClickReaction(ev);
                 return;
             }
-            if (this.emojiGridItemViewOwner.emojiGridRowViewOwner.emojiGridViewOwner.emojiPickerViewOwner.popoverViewOwner.composerViewOwnerAsEmoji) {
-                this.emojiGridItemViewOwner.emojiGridRowViewOwner.emojiGridViewOwner.emojiPickerViewOwner.popoverViewOwner.composerViewOwnerAsEmoji.onClickEmoji(ev);
+            if (this.popoverViewTopOwner.composerViewOwnerAsEmoji) {
+                this.popoverViewTopOwner.composerViewOwnerAsEmoji.onClickEmoji(ev);
```

Comment:
You could also make this change in other modules, such as `knowledge` and `social`.
That way if there's a need to use Emoji Picker in another context, we guarantee the copying is nicer to read and write ;)